### DISCUSSION TITLE
Cant get "Evaluate MIDI track" to work.

### DISCUSSION THREAD

**Discussion by ColePeterson** [JacquesLucke/animation_nodes#1903] [Q&A]
I've attempted to follow the MIDI tutorial here: https://docs.animation-nodes.com/tutorials/midi/ and everything seems to work except the Evaluate MIDI track node. I may just have a misunderstanding of what it does.
I can view the tracks and midi notes with the viewer but no matter what I've tried, Evaluate MIDI track always outputs zero.

I'm on Blender 3.4.1 and animation nodes 2.3. I apologize if this is a known issue with these versions, I wasn't able to find much online.
Thank you!

My test setup:
![image](https://user-images.githubusercontent.com/25966938/216884025-f019fca2-3b2e-4a49-bb11-0f34e90b40eb.png)

**Comment by OmarEmaraDev** [comment]
Are you sure it is not working? In your screenshot, you are looking at only the first 14 notes, which are unlikely to contain notes.
The notes you are viewing are the 24th and 36th notes, so maybe look at the values for those at the right times.
